packages/client/src/components/ProgressWidget/ProgressWidget.tsx

```diff
@@ -11,14 +11,19 @@ const myPathsQuery = gql`query {
   myPaths { id name icon }
 }`;
 
-export interface ProgressWidgetProps extends CardProps { }
+export interface ProgressWidgetProps extends CardProps {
+  setShow: any;
```

Comment:
Is this always a boolean? If yes, maybe we should change the type here.

Reply:
setShow is a function and I can't figure out how to type it